pallets/creditcoin/src/migrations/v5.rs

```diff
@@ -68,47 +69,68 @@ impl<Hash> From<CollectedCoinsId<Hash>> for TaskId<Hash> {
 	}
 }
 
-generate_storage_alias!(
-	Creditcoin,
-	UnverifiedTransfers<T: Config> => DoubleMap<
-		(Identity, T::BlockNumber),
-		(Identity, TransferId<T::Hash>),
-		UnverifiedTransfer<T::AccountId, T::BlockNumber, T::Hash, T::Moment>
-	>
-);
-
-generate_storage_alias!(
-	Creditcoin,
-	UnverifiedCollectedCoins<T: Config> => DoubleMap<
-		(Identity, T::BlockNumber),
-		(Identity, CollectedCoinsId<T::Hash>),
-		UnverifiedCollectedCoinsStruct
-	>
-);
-
-generate_storage_alias!(
-	Creditcoin,
-	PendingTasks<T: Config> => DoubleMap<
-		(Identity, T::BlockNumber),
-		(Identity, TaskId<T::Hash>),
-		Task<T::AccountId, T::BlockNumber, T::Hash, T::Moment>
-	>
-);
+// generate_storage_alias!(
+// 	Creditcoin,
+// 	UnverifiedTransfers<T: Config> => DoubleMap<
+// 		(Identity, BlockNumberOf<T>),
+// 		(Identity, TransferId<HashOf<T>>),
+// 		UnverifiedTransfer<AccountIdOf<T>, BlockNumberOf<T>, HashOf<T>, MomentOf<T>>
+// 	>
+// );
+
+#[storage_alias]
```

Comment:
It seems very redundant to have two new macros for storage aliases plus another `storage_alias` macro attribute. We should stick with a single approach. Also, there is a lot of commented out left overs.

Reply:
I expanded a bit in [this comment](https://github.com/gluwa/creditcoin/pull/801#discussion_r1041692976), but yeah it might not be worth the cognitive overhead just to avoid the duplication. I pushed a commit to remove `storage_macro`, here's [the diff](https://github.com/gluwa/creditcoin/pull/801/commits/c1ac0a0864dd52004e11dd3b6495d3412650a420)